pupa/scrape/base.py

```diff
@@ -136,6 +138,16 @@ def scrape(self, legislative_session, **kwargs):
                 continue
 
 
+class BaseDisclosureScraper(Scraper):
+
+    def do_scrape(self, **kwargs):
+        record = super(BaseDisclosureScraper, self).do_scrape(**kwargs)
+        utils.makedirs(settings.PERSIST_DATA_DIR)
+        for f in glob.glob(settings.SCRAPED_DATA_DIR + '/*/*.json'):
+            shutil.copy(f, settings.PERSIST_DATA_DIR)
+        return record
```

Comment:
Hurm, can we either rename this to be a bit more meaningful to other scrapers, or move it into the same repo as the other `Disclosure` stuff?